src/package/packageCreate.ts

```diff
@@ -47,7 +47,7 @@ export function createPackageDirEntry(project: SfProject, options: PackageCreate
   if (!packageDir) {
     // no match - create a new one
     isNew = true;
-    packageDir = pkgUtils.DEFAULT_PACKAGE_DIR as NamedPackageDir;
+    packageDir = { ...pkgUtils.DEFAULT_PACKAGE_DIR } as NamedPackageDir;
```

Comment:
This fixes a bug in the test harness whereby `pkgUtils.DEFAULT_PACKAGE_DIR` could be altered by tests and cause subsequent tests to fail.